website/redirects.js

```diff
@@ -50,8 +50,8 @@ module.exports = [
     permanent: true,
   },
   {
-    source: '/docs/auth/jwt_oidc_providers',
```

Comment:
I think we should keep this redirect (instead of editing) and add a new redirect? i.e. `/docs/auth/jwt_oidc_providers` -> `/docs/auth/jwt/oidc-providers` and `/docs/auth/jwt/oidc_providers` -> `/docs/auth/jwt/oidc-providers`

Reply:
Ah, yes. That seems like the right thing to do. I've updated the redirects in 4018e0b. Thanks!